core/arch/arm32/kernel/sub.mk

```diff
@@ -1,7 +1,7 @@
 srcs-y += tee_ta_manager.c
 srcs-y += tee_time.c
 
-srcs-$(WITH_SECURE_TIME_SOURCE_CNTPCT) += tee_time_arm_cntpct.c
+srcs-$(WITH_SECURE_TIME_SOURCE_CNTPCT) += tee_time_arm_generic_timer.c
```

Comment:
WITH_SECURE_TIME_SOURCE_CNTPCT could be renamed in WITH_SECURE_TIME_GENERIC_TIMER